### PR TITLE
updated metadata for v20.8 models

### PR DESCRIPTION
**Pull request by vivekjhauipath**

### CODE REVIEW

File: metadata/InvoicesAustralia__2__metadata.json [hunk 1]
```diff
@@ -17,5 +17,5 @@
   "projectName": "UiPath Document Understanding",
   "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
   "tenantName": "UiPath",
-  "imagePath": "registry.replicated.com/aif-core/australianinvoices:2"
+  "imagePath": "registry.replicated.com/aif-core/invoicesaustralia:2"
```
Comment (mohitseth-nitrr) [inline metadata/InvoicesAustralia__2__metadata.json:20]: Is this change required ? Images are already pushed

File: metadata/InvoicesAustralia__1__metadata.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "changeLog": "",
   "cpu": 0,
   "description": "Machine Learning model for extracting commonly occurring data points from Australian Invoices, including header fields and line items. Please see more details including supported languages and link to Activities guide in the About Licensing -> Document Understanding API Key section of the UiPath Automation Cloud Guide here: https://docs.uipath.com/automation-cloud/docs/about-licensing#section-document-understanding-api-key",
-  "displayName": "AustralianInvoices",
+  "displayName": "InvoicesAustralia",
```
Comment (marcoalban-hidalgo) [inline metadata/InvoicesAustralia__1__metadata.json:5]: "name" and "displayName" should match no? "name" also needs to change to "InvoicesAustralia"

File: metadata/InvoicesAustralia__2__metadata.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
-  "changeLog": "v20.8 release",
+  "changeLog": "Release v2020.8",
   "cpu": 0,
   "description": "Machine Learning model for extracting commonly occurring data points from Australian Invoices, including header fields and line items. Please see more details including supported languages and link to Activities guide in the About Licensing -> Document Understanding API Key section of the UiPath Automation Cloud Guide here: https://docs.uipath.com/automation-cloud/docs/about-licensing#section-document-understanding-api-key",
-  "displayName": "AustralianInvoices",
+  "displayName": "InvoicesAustralia",
```
Comment (marcoalban-hidalgo) [inline metadata/InvoicesAustralia__2__metadata.json:5]: "name" and "displayName" should match no? "name" also needs to change to "InvoicesAustralia"

File: metadata/InvoicesIndia__1__metadata.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "changeLog": "",
   "cpu": 0,
   "description": "Machine Learning model for extracting commonly occurring data points from Indian Invoices, including header fields and line items. Please see more details including supported languages and link to Activities guide in the About Licensing -> Document Understanding API Key section of the UiPath Automation Cloud Guide here: https://docs.uipath.com/automation-cloud/docs/about-licensing#section-document-understanding-api-key.",
-  "displayName": "IndianInvoices",
+  "displayName": "InvoicesIndia",
```
Comment (marcoalban-hidalgo) [inline metadata/InvoicesIndia__1__metadata.json:5]: "name" and "displayName" should match, "name" also needs to change to "InvoicesIndia"

File: metadata/InvoicesIndia__2__metadata.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  "changeLog": "Release v2020.8",
+  "cpu": 0,
+  "description": "Machine Learning model for extracting commonly occurring data points from Indian Invoices, including header fields and line items. Please see more details including supported languages and link to Activities guide in the About Licensing -> Document Understanding API Key section of the UiPath Automation Cloud Guide here: https://docs.uipath.com/automation-cloud/docs/about-licensing#section-document-understanding-api-key.",
+  "displayName": "InvoicesIndia",
+  "gpu": 0,
+  "inputDescription": "ML Skills deployed using this ML Package can be integrated into RPA workflows using the Machine Learning Extractor activity in the Official feed or the MLModel activity in Connect feed. File formats accepted include pdf, tiff, jpg or png files. The activities also require an API Key input which you need to obtain from your UiPath Automation Cloud account, in the Licenses -> Other Services view.",
+  "inputType": "JSON",
+  "memory": 0,
+  "mlPackageLanguage": "PYTHON37_DU",
+  "name": "IndianInvoices",
```
Comment (marcoalban-hidalgo) [inline metadata/InvoicesIndia__2__metadata.json:11]: "name" = "InvoicesIndia"

File: metadata/InvoicesAustralia__2__metadata.json (hunk 1 above)
Comment (mohitseth-nitrr) [reply metadata/InvoicesAustralia__2__metadata.json:20]: Also, 1st version might already be present in v20.7.2 release

Comment (vivekjhauipath) [reply metadata/InvoicesAustralia__2__metadata.json:20]: We can retag and push.
We are changing modelNames to match those in prod for:
IndianInvoices => InvoicesIndia
AustralianInvoices => InvoicesAustralia.

Any concerns with this change? Will we need to delete in db?

Comment (vivekjhauipath) [reply metadata/InvoicesAustralia__2__metadata.json:20]: @mohitseth-nitrr

File: metadata/InvoicesAustralia__1__metadata.json (hunk 2 above)
Comment (vivekjhauipath) [reply metadata/InvoicesAustralia__1__metadata.json:5]: good call, thanks

File: metadata/InvoicesAustralia__2__metadata.json (hunk 1 above)
Comment (mohitseth-nitrr) [reply metadata/InvoicesAustralia__2__metadata.json:20]: we need to make many changes ... and provide a upgrade path ... seems like too much work here

**Review by mohitseth-nitrr — APPROVED** [review]
LGTM

**Review by aakashuipath — APPROVED** [review]
